### PR TITLE
feat: add link and and sub-section for related resources

### LINKED ISSUE #904: Update `/documentation#resources`
**Please describe the feature**

As a user, I want the resources section on the Documentation page to be up to date so that I can easily find what I'm looking for.

**Describe the solution you would like**

A link to `/about/resources`.

**Additional context**

See https://github.com/MetabolicAtlas/MetabolicAtlas/pull/902#pullrequestreview-904290862

The outdated documentation is confusing for users.

### PR DESCRIPTION
**Pull request by e0**
**Related issue(s) and PR(s)**  
This PR closes #904.

**Type of change**  

- [x] Other
 
**List of changes made**  

Added a link and subsection for related resources on the documentation page.

**Screenshot of the fix**  

image

**Testing**  

- Relative urls that can be reused both for production and local testing
- Instructions on how to test
  1. Visit `/documentation#related-resources`.
  2. Click the `about page section` link.
  3. Verify that the `/about/resources` page loads.

**Further comments**  

This satisfies the acceptance criteria but does not address [this suggestion](https://github.com/MetabolicAtlas/MetabolicAtlas/issues/904#issuecomment-1062926745). Personally I don't feel that the are necessary although they could be nice to have. Please feel free to add the changes directly to this branch or leave a comment with the proposed changes/additions.

**Definition of Done checklist**  
- [x] My code meets the Acceptance Criteria
- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code and commented any hard-to-understand areas
- [x] Tests and lint/format validations are passing
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by inghylt — APPROVED** [review]
Nice! I like the addition with a new header

**Review by MalinAhlberg — APPROVED** [review]
Nice, this makes the section much clearer!

**Comment by MalinAhlberg** [comment]
> This satisfies the acceptance criteria but does not address this suggestion.

One thing that might improve the `Related resources` section (under about) is to add  subsections in the left menu (Tools, Databases, APIs):
subsection-resources2

A bit unrelated to this PR, anyway.

File: frontend/src/components/Documentation.vue [hunk 1]
```diff
@@ -616,6 +611,12 @@
             10 requests per second. The API is still undergoing development; while it is covers the
```
Comment (nanjiangshu) [inline frontend/src/components/Documentation.vue:611]:
```suggestion
            10 requests per second. The API is still undergoing development; while it covers the
```

**Review by nanjiangshu — COMMENTED** [review]
Well written @e0!

Just a minor comment for the grammatically change.

There is another minor thing which also has text in this section but may not be directly related to the related resource. 

On line https://github.com/MetabolicAtlas/MetabolicAtlas/blob/feat/related-resources-link/frontend/src/components/Documentation.vue#L608, the url of the API is set to `/api/v2`. Since we already has a redirect of `/api` -> /api/v2`. I think it might be more natural to change it to `/api` here. One benefit is that if we later one changed the re-direct to `/api -> /api/v3`, the link in this documentation does not need to be changed. 

If you agree to change it, might be less overhead to change it here than to create a new issue for it.

File: frontend/src/components/Documentation.vue (hunk 1 above)
Comment (e0) [reply frontend/src/components/Documentation.vue]: Thanks!

**Comment by e0** [comment]
I really like the suggestions by @MalinAhlberg and @nanjiangshu, to add the sub-sections and change the link. Happy to include them in this PR. Any thoughts on this @inghylt?

**Comment by e0** [comment]
> I really like the suggestions by @MalinAhlberg and @nanjiangshu, to add the sub-sections and change the link. Happy to include them in this PR. Any thoughts on this @inghylt?

It seems like we have several positive sentiments about these changes so I had added some commits for them now. Just want to re-request some reviews to make sure we are all on the same page.

**Review by nanjiangshu — APPROVED** [review]
Looks great with the new changes you added!

**Review by MalinAhlberg — APPROVED** [review]
Really nice with the new changes!

**Pull request merged**